src/services/Queries.js

```diff
@@ -299,5 +320,95 @@ class Queries {
 
     return false;
   }
+
+  /**
+   * find trip function
+   * @param {String} table table name
+   * @param {Object} tripId trip id
+   * @returns { Object } result
+   */
+  static async getTripRequestByTripId(table, tripId) {
+    const data = await table.findAll({
+      where: {
+        tripId,
+      }
+    });
+    return data;
+  }
+
+  /**
+   *
+   * Function to update the status to approved or reject
+   * @param {String} table the name of the table to updated
+   * @param {string} tripId the id of the selected trip
+   * @param {Object} data it gets the data from req.body
+   * @returns {object} updated data
+   */
+  static async updateTrip(table, tripId, data) {
+    try {
+      const updatedRequest = await table.update(
+        data,
+        { where: { tripId } }
+      );
+      return updatedRequest;
+    } catch (error) {
+      return error;
+    }
+  }
+
+  /**
+   *
+   * Function to update the status to approved or reject
+   * @param {String} table the name of the table to updated
+   * @param {string} tripId the id of the selected trip
+   * @param {Object} data it gets the data from req.body
+   * @returns {object} updated data
+   */
+  static async updateMultiCityTripRequest(table, tripId, data) {
+    try {
+      const updatedRequest = await table.update(
+        data,
+        { where: { id: tripId } }
+      );
+      return updatedRequest;
+    } catch (error) {
+      return error;
+    }
+  }
+
+  /** Function to update the status to approved or reject
+   *
+   * @param {object} table to be updating in
+   * @param {integer} id trip id to be found and then replace the status
+   * @returns {object} updated data
+   */
+  static async updateRequestStatusById(table, id) {
+    try {
```

Comment:
Hey, @mujohn26 you did a good job so far, however, there is something you can change.
`updateRequestStatusById`, `updateMultiCityTripRequest` and `updateTrip`  methods are doing the same thing, so you can find a way of using a single method, and think that can be better.